samples/bluetooth/central_hr/src/main.c

```diff
@@ -180,6 +180,20 @@ static void device_found(const bt_addr_le_t *addr, s8_t rssi, u8_t type,
 	}
 }
 
+static int scan_start(void)
+{
+	/* Use active scanning and disable duplicate filtering to handle any
+	 * devices that might update their advertising data at runtime. */
+	struct bt_le_scan_param scan_param = {
+		.type       = BT_HCI_LE_SCAN_ACTIVE,
+		.filter_dup = BT_HCI_LE_SCAN_FILTER_DUP_DISABLE,
```

Comment:
I find it strange that this would be fixed for only this sample. If this is a normal thing that could happen shouldn't we set it for all samples?

Possibly add a define for it in bluetooth.h?